src/context/init.js

```diff
@@ -52,6 +53,7 @@ function initServices(context) {
   context.addInstance('errorHandler', errorHandler);
   context.addInstance('ipWhitelist', ipWhitelist);
   context.addInstance('forestServerRequester', forestServerRequester);
+  context.addInstance('errorService', errorService);
```

Comment:
[Suggestion] Shouldn't this also be a part of the type ?
(I totally forgot this on my first PRs, but I guess since it is in the code now, adding it would be nice)